cli/src/supervisorCommands.js

```diff
@@ -0,0 +1,119 @@
+/* Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * See AUTHORS.txt
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ * This file is part of the OperatorFabric project.
+ */
+
+const prompts = require('prompts');
+const utils = require('./utils.js');
+
+const supervisorCommands = {
+    async processSupervisorCommand(args) {
+        let command = args[0];
+        if (!command) {
+            command = (
+                await prompts({
+                    type: 'select',
+                    name: 'value',
+                    message: 'Supervisor command',
+                    choices: [
+                        {title: 'add supervising entity', value: 'add-entity'},
+                        {title: 'delete supervising entities', value: 'delete-entity'}
+                    ]
+                })
+            ).value;
+            if (!command) {
+                console.log('Supervisor command is required');
+                return;
+            }
+        }
+
+        switch (command) {
+            case 'add-entity':
+                await this.addSupervisorEntity(args[1], args[2]);
+                break;
+            case 'delete-entity':
+                await this.deleteSupervisedEntity(args[1]);
+                break;
+            default:
+                console.log(`Unknown supervisor command : ${command}
+                `);
+                await this.printHelp();
+                break;
+        }
+    },
+
+    async addSupervisorEntity(supervisedEntity, supervisingEntity) {
+        supervisedEntity = await utils.missingTextPrompt('Supervised entity name', supervisedEntity);
```

Comment:
Maybe it is better to check if supervisedEntity was set and if was not set then exit without asking for supervisor.

Reply:
Done